graphql-dgs-spring-webmvc-autoconfigure/src/test/kotlin/com/netflix/graphql/dgs/webmvc/autoconfiguration/DgsWebMvcAutoConfigurationTest.kt

```diff
@@ -43,86 +44,117 @@ import org.springframework.test.web.servlet.setup.MockMvcBuilders
 
 class DgsWebMvcAutoConfigurationTest {
 
-    private val context = WebApplicationContextRunner().withConfiguration(
-        AutoConfigurations.of(DgsWebMvcAutoconfiguration::class.java, MockAutoConfiguration::class.java))!!
+	private val context = WebApplicationContextRunner().withConfiguration(
+			AutoConfigurations.of(DgsWebMvcAutoconfiguration::class.java, MockAutoConfiguration::class.java))!!
 
-    @Test
-    fun graphqlControllerAvailable() {
-        context.run { ctx ->
-            Assertions.assertThat(ctx).hasSingleBean(DgsRestController::class.java)
-        }
-    }
+	@Test
+	fun graphqlControllerAvailable() {
+		context.run { ctx ->
```

Comment:
@paulbakker my formatting is off. I don't think its picking up what I thought it was. 

Here is my IDEA info (I bet I need to update). When I tried to import the schema it complained about improper configuration root element. 
```java
IntelliJ IDEA 2020.1.4 (Ultimate Edition)
Build #IU-201.8743.12, built on July 21, 2020
Licensed to Indeed / Chris Bono
You have a perpetual fallback license for this version
Subscription is active until November 6, 2021
Runtime version: 11.0.7+10-b765.65 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 10.15.7
GC: ParNew, ConcurrentMarkSweep
Memory: 7891M
Cores: 16
Non-Bundled Plugins: CheckStyle-IDEA, Lombook Plugin, com.chainstaysoftware.assertions2assertj, com.bluetrainsoftware.groovy.idea, Pythonid, org.jetbrains.kotlin, org.intellij.scala, org.toml.lang
```

Reply:
I updated to EAP and the formatting does look better. 
```java
IntelliJ IDEA 2021.1 EAP (Ultimate Edition)
Build #IU-211.6222.4, built on February 25, 2021
IntelliJ IDEA EAP User
Expiration date: March 27, 2021
Runtime version: 11.0.10+9-b1341.2 x86_64
VM: Dynamic Code Evolution 64-Bit Server VM by JetBrains s.r.o.
macOS 10.15.7
GC: G1 Young Generation, G1 Old Generation
Memory: 8000M
Cores: 16
Non-Bundled Plugins: org.toml.lang (0.2.142.3705-211), CheckStyle-IDEA (5.47.0), com.chainstaysoftware.assertions2assertj (1.0.05), com.bluetrainsoftware.groovy.idea (1.0), org.intellij.scala (2021.1.11), Pythonid (211.6222.4)
Kotlin: 211-1.4.21-release-IJ6222.3
```

Reply:
Interesting, I'm actually not on EAP at the moment, just 2020.3.2. Hopefully it works consistently.

Reply:
I think I found the issue; I had tried to share the Intellij configuration but had done so incorrectly, so it would still pick your global IDE settings. I've updated it, and it should now work across versions.

Reply:
> so it would still pick your global IDE settings

cool. yeh this is exactly what it looked like it was doing. I ended up setting it manually to get it where it was using the correct settings. 

I just now pulled/rebased and IDEA opened a little popup asking:
```java
Do you want to update your Kotlin code style? 
```
looks like it's working now. I'll keep my eye on it as I go in case I see anything strange etc..